### PR TITLE
Compliance shouldn't fail hard if notifications is down

### PR DESCRIPTION
**Pull request by jaym**

### CODE REVIEW

**Comment by stevendanna** [comment]
Might be useful to include some words about why we don't need to block startup for this service in particular.

**Comment by lancewf** [comment]
What happens when the notification service is not there and a failed InSpec report is ingested?

**Comment by stevendanna** [comment]
> What happens when the notification service is not there and a failed InSpec report is ingested?

I don't think blocking here prevents you from having to handle that case though. notifications-service could go away at any time.

File: components/compliance-service/compliance.go [hunk 1]
```diff
@@ -262,7 +262,7 @@ func getNotificationsConnection(connectionFactory *secureconn.Factory,
 	timeoutCtx, cancel := context.WithTimeout(context.Background(), 60*time.Second)
 	defer cancel()
 	conn, err := connectionFactory.DialContext(timeoutCtx, "notifications-service",
```
Comment (lancewf) [inline components/compliance-service/compliance.go]: If the notification-service is not running, will this wait until the 60-second timeout and return a connection `conn` without an error?

Comment (jaym) [reply components/compliance-service/compliance.go]: i don't think so. I removed the WithBlock so it doesn't wait for anything

**Pull request closed without merging**